plugin/pkg/scheduler/algorithm/priorities/image_locality.go

```diff
@@ -31,10 +29,6 @@ import (
 // - If some of the images are present on a node, the larger their sizes' sum, the higher the node's priority.
 func ImageLocalityPriorityMap(pod *v1.Pod, meta interface{}, nodeInfo *schedulercache.NodeInfo) (schedulerapi.HostPriority, error) {
 	node := nodeInfo.Node()
-	if node == nil {
```

Comment:
I think we should keep the checks in priority functions. These functions may be called by external or custom schedulers and generic_scheduler is not the only user of these. There may already be custom schedulers that rely on the fact that these functions handle nil nodes correctly and removing the check may break them.

Reply:
Ops, I don't know that. I thought genericSchduler is the only implementation of `SchedulerAlgorithm`. Can you point me to the Custom schedulers?

Also, in that case, let's define an ErrNotFound, so that we don't worry about the wording.

Reply:
Custom schedulers are schedulers that Kubernetes customers build for themselves. Their code is not located in our tree.

Reply:
I agree we Bobby. Closing this PR.